### DISCUSSION TITLE
IOS微信中使用直播flv无法正常播放

### DISCUSSION THREAD

**Discussion by zhanhy2015** [bytedance/xgplayer#1107] [Q&A]
IOS微信中使用直播flv无法正常播放，使用案例放在ios微信中也无法播放，这个问题怎么解决呢。其它人有同样的情况吗？

https://github.com/bytedance/xgplayer/assets/145098374/231b4120-d9c4-45b6-ae8f-4461c9e01993

**Comment by gemxx** [comment]
iOS 17之前还不支持MSE，因此无法支持播放Flv
